keyboards/silverbullet44/keymaps/FKML/keymap.c

```diff
@@ -0,0 +1,148 @@
+  *                            |      |      |      /       /     \       \     |      |      |
+ *                             `---------------------------'       '--------------------------'
+ */
+    [_RAISE] = LAYOUT( 
+  _______,   KC_F1,   KC_F2,    KC_F3,   KC_F4,  KC_F5,                     _______, _______, _______, _______,  KC_MINS, KC_PLUS, 
+  _______,   KC_F6,   KC_F7,    KC_F8,   KC_F9,  KC_F10,                    _______, _______, _______, _______,  KC_LBRC, KC_RBRC, 
+  _______,  KC_F11,   KC_F12, _______, _______, _______,                    _______, _______, KC_LEFT,   KC_UP,  KC_DOWN, KC_RGHT, 
+                              _______, _______, _______, _______,   ADJUST, _______, _______, _______ 
+  ),
+
+/*   ADJUST
+ * ,-----------------------------------------.                                  ,-----------------------------------------.
+ * |      |      |      |FrqRst|      |      |                                  |      |      |      |      |      |      |
+ * |------+------+------+------+------+------|                                  |------+------+------+------+------+------|
+ * |      |      |Music |FrqUp |Sound |      |                                  | SPD+ | MOD+ | VAL+ | SAD+ | HUE+ |      |
+ * |------+------+------+------+------+------|                                  |------+------+------+------+------+------|
+ * |      |      |Scale |FrqDwn|      |      |-------.-------.  ,---------------| SPD- | SPD- | VAL- | SAD- | HUE- |      |
+ * `-----------------------------------------/       /       /   \       \      \-----------------------------------------'
+ *                             |      |      |      /       /     \       \     |      |      |
+ *                             `---------------------------'       '--------------------------'
+*/
+
+ [_ADJUST] = LAYOUT( 
+  _______, XXXXXXX, XXXXXXX, CK_RST,  XXXXXXX, XXXXXXX,                      XXXXXXX, XXXXXXX,  XXXXXXX, XXXXXXX, XXXXXXX, _______, 
+  XXXXXXX, XXXXXXX, MU_TOG,  CK_UP,   AU_TOG,  XXXXXXX,                      RGB_SPI, RGB_MOD,  RGB_VAI, RGB_SAI, RGB_HUI, XXXXXXX, 
+  XXXXXXX, XXXXXXX, MU_MOD,  CK_DOWN, XXXXXXX, XXXXXXX,                      RGB_SPD, RGB_RMOD, RGB_VAD, RGB_SAD, RGB_HUD, XXXXXXX, 
+                             XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX,    XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX 
+  )
+};
+
+
+#ifdef RGBLIGHT_ENABLE
+//Following line allows macro to read current RGB settings
+extern rgblight_config_t rgblight_config;
+#endif
+
+int RGB_current_mode;
+
+bool process_record_user(uint16_t keycode, keyrecord_t *record) {
+  switch (keycode) {
+      case LOWER:
+        if (record->event.pressed) {
+            layer_on(_LOWER);
+            update_tri_layer(_LOWER, _RAISE, _ADJUST);
+        } else {
+            layer_off(_LOWER);
+            update_tri_layer(_LOWER, _RAISE, _ADJUST);
+        }
+        return false;
+        break;
+        case RAISE:
```

Comment:
インデント
あと、ここだけインデントが2spaceになってるので
4spaceにしとくと良さそう。

```suggestion
      case RAISE:
```